### ISSUE TITLE
Adding an webhook integration to a sequence blocks just spins

### ISSUE THREAD

**Issue by ropelletier** [chiefonboarding/ChiefOnboarding#176]
I created a test integration webhook, but when I attempt to add this to a sequence block it just shows a dialog box with Item header and the spinning wheel. Clicking submit does nothing. Logs do not show anything. 

![image](https://user-images.githubusercontent.com/88781432/178267266-5f5e098e-447e-41cf-b327-7b2828c0b267.png)

**Comment by GDay** [comment]
Can you please share the config that you are using (the json manifest)? Please change any urls or keys/secrets (keys/secrets shouldn't be in the config anyway).

**Comment by ropelletier** [comment]
Of course. To explain, All I want to do is send a json payload with the employee data via POST to an internal website. I have a php target file that processes the data. I took a stab in a dark regarding how the manifest should look. Not sure what, if anything is required. I kept the form data almost entirely the same as your example just for now. This was meant to simply test if the employees email address was properly sent in the POST payload. 


{"form": [{"id": "TEAM_ID", "url": "https://psapi.rsu87.org/jamf_endpoint.php", "name": "Select team to add user to", "type": "choice", "data_from": "data", "choice_name": "name", "choice_value": "gid"}], "execute": [{"url": "https://psapi.rsu87.org/jamf_endpoint.php", "data": {"data": {"user": "{{email}}"}}, "method": "POST"}], "headers": {"Accept": "application/json", "Content-Type": "application/json", "Authorization": "Bearer 123456789"}}

**Comment by GDay** [comment]
Okay, thanks. So that form URL should respond with something like this to be processed correctly:

```
{
  "data": [
    {
      "gid": "12345",
      "name": "Marketing"
    },
    {
      "gid": "12345",
      "name": "Marketing"
    }
  ]
}
```
Does your url do that? I assume it didn't.

Error handling/showing is a little lacking at the moment, will fix that later today.

**Comment by GDay** [comment]
Oh wait, I get the same thing now with the default json manifest. Will get fixed asap!

**Comment by GDay** [comment]
Oh, nevermind my last comment. It works for asana (I just put in the wrong secrets in my staging environment, lol). But error handling should be better and will fix that as mentioned above later today.

**Comment by ropelletier** [comment]
Ahhhh, now I understand. That's all I needed. Working now. Thank you.

**Comment by GDay** [comment]
There is now a much better error handling for when things don't match up:

![image](https://user-images.githubusercontent.com/1939656/178379568-689de66e-beeb-424e-bb3e-40280e4c4f7a.png)

Fixed by merging PR:  #181
